### PR TITLE
Update 20. Valid Parentheses.cpp

### PR DESCRIPTION
**Pull request by md-shamim-ahmad**

### CODE REVIEW

**Review by ghost — CHANGES_REQUESTED** [review]
Hi,
It will be good if you add the question readme.md file for this problem

File: LeetCode Problems/1 to 100/20. Valid Parentheses/20. Valid Parentheses.cpp [hunk 1]
```diff
@@ -1,23 +1,17 @@
...
-            else if (s[i] == '}' && arr.top() == '{')
-                arr.pop();
-            else if (s[i] == ']' && arr.top() == '[')
-                arr.pop();
-            else 
-                return false;
+            if(stc.empty()) return false;
+            else if (s[i] == ')' && stc.top() != '(') return false;                
+            else if (s[i] == '}' && stc.top() != '{') return false;
+            else if (s[i] == ']' && stc.top() != '[') return false;
```
Comment (ghost) [inline LeetCode Problems/1 to 100/20. Valid Parentheses/20. Valid Parentheses.cpp:12]: can we reduce those code :
```
else if (s[i] == ')' && stc.top() != '(') return false;                
else if (s[i] == '}' && stc.top() != '{') return false;
else if (s[i] == ']' && stc.top() != '[') return false;
```
as 
```
else if ( 
             (s[i] == ')' && stc.top() != '(') || 
             (s[i] == '}' && stc.top() != '{') || 
             (s[i] == ']' && stc.top() != '[')
)  return false;
```

**Review by ghost — CHANGES_REQUESTED** [review]

File: LeetCode Problems/1 to 100/20. Valid Parentheses/20. Valid Parentheses.cpp (hunk 1 above)
Comment (md-shamim-ahmad) [reply LeetCode Problems/1 to 100/20. Valid Parentheses/20. Valid Parentheses.cpp:12]: Hi,
thanks for your  suggested.
But I think they are closely the same.
So no need to change.

**Comment by md-shamim-ahmad** [comment]
> Hi, It will be good if you add the question readme.md file for this problem
thanks for your comment ...
[check it now](https://github.com/Md-Shamim-Ahmmed/LeetCode-Problem-Solution/tree/main/LeetCode%20Problems/1%20to%20100/20.%20Valid%20Parentheses)

**Pull request merged**